emcc

```diff
@@ -109,7 +109,8 @@ stderr = PIPE if not DEBUG else None # unless we are in DEBUG mode
 EMCC_CXX = '--emscripten-cxx' in sys.argv
 sys.argv = filter(lambda x: x != '--emscripten-cxx', sys.argv)
 
-shared.check_sanity(force=DEBUG)
+if len(sys.argv) != 1 and '--help' not in sys.argv and sys.argv[1] != '--version':
+  shared.check_sanity(force=DEBUG)
```

Comment:
Hmm, I deliberately allowed these because I imagined "emscripten won't start, I need to downgrade! I wonder what version I have..." or "I remember a list of environment variables saying something about fastcomp, let's look at the help".
You can still trigger a sanity check with `emcc -v`.